packages/ui-date-time-input/src/DateTimeInput/index.tsx

```diff
@@ -515,7 +515,8 @@ class DateTimeInput extends Component<DateTimeInputProps, DateTimeInputState> {
       colSpacing,
       isRequired,
       interaction,
-      renderWeekdayLabels
+      renderWeekdayLabels,
+      allowNonStepInput
```

Comment:
I would add `allowNonStepInput: false` to the defaultProps.

Reply:
fixed